### DISCUSSION TITLE
How to implement cross-validation custom evaluation functions

### DISCUSSION THREAD

**Discussion by dengxuezhao** [winedarksea/AutoTS#183] [Q&A]
Hello, my data is daily frequency data, now I want to predict the data for the next three months and then aggregate the prediction results to the monthly frequency calculation evaluation index during cross-validation, how do I need to change the code to achieve my effect?

**Comment by winedarksea** [comment]
You should aggregate the data to monthly before feeding it into the model. That is the best way to assure it aligns as desired. Also, accuracy metrics look at all individual data points, so it is always best to do everything at the level of the desired predictions. Look at pandas reindex.

In AutoTS you can specify `frequency='MS'` for month start and `aggfunc='mean'` or 'sum' or 'first' or 'last' to automatically aggregate
